utils/openshift_resource.py

```diff
@@ -406,16 +406,24 @@ def initialize_resource_type(self, cluster, namespace, resource_type):
 
     def add_desired(self, cluster, namespace, resource_type, name, value):
         with self._lock:
-            desired = \
-                self._clusters[cluster][namespace][resource_type]['desired']
+            try:
```

Comment:
why do we need this?

Reply:
I think we never get to run this if the `oc_map.get(cluster)` gives us an `OcLogMsg` instance.

Reply:
`openshift-saas-deploy-wrapper` integration fails without it.  I will retest without the try/except block, but I got hung up here with an exception until I added this.

Reply:
maybe that was before all the other fixes? yes, check it with all the other fixes here in place please.

Reply:
:+1: